presidio-anonymizer/README.MD

```diff
@@ -5,6 +5,8 @@
 The Presidio anonymizer is a Python based module for anonymizing detected PII text
 entities with desired values.
 
+![Anonymizer Design](../docs/assets/anonymizer-design.png)
```

Comment:
Both here, and for the image redactor, should be change to '../assets/analyzer-design.png' as for the analyzer, after the docs will move the docs dir

Reply:
why cant this be done now?

Reply:
It can, understood from @shiranr in the daily that shes taking it?